x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/add_data_provider_popover.tsx

```diff
@@ -71,14 +71,14 @@ const AddDataProviderPopoverComponent: React.FC<AddDataProviderPopoverProps> = (
         onAddedToTimeline: handleClosePopover,
         providerToAdd: {
           id: providerId,
-          name: value,
+          name: field,
```

Comment:
The name field expects a string, and value can now be of type Array, causing an error with persistence.